R/colour-text.r

```diff
@@ -5,6 +5,7 @@ testthat_colours <- list(
   error = crayon::red
 )
 
+#' @export
 colourise <- function(text, as = c("passed", "skipped", "error")) {
```

Comment:
How's this?

On Wed, Jan 7, 2015 at 4:26 PM, Kirill Müller <notifications@github.com>
wrote:

> In R/colour-text.r
> <https://github.com/hadley/testthat/pull/211#discussion-diff-22621607>:
>
> > @@ -5,6 +5,7 @@ testthat_colours <- list(
> >    error = crayon::red
> >  )
> >
> > +#' @export
> >  colourise <- function(text, as = c("passed", "skipped", "error")) {
>
> I think the parameters fg and bg should remain and trigger a warning when
> used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hadley/testthat/pull/211/files#r22621607>.
>



-- 
-Robert Krzyzanowski